misc-tools/dj_judgehost_cleanup.in

```diff
@@ -90,6 +86,32 @@ case $1 in
             cd - >/dev/null
         done
         echo "Done."
+}
+
+if [ "$(id -u)" -ne 0 ]; then
+    echo "This program must be run as root."
+    echo
+    usage
+    exit 1
+fi
+
+case $1 in
+    mount*)
+	cleanup_mounts
```

Comment:
Nitpick: I'd say indentation should be one level deeper here.

Reply:
It currently is a mix of tabs and spaces, sometimes 4, sometimes 8.

We probably want to make it consistent. What indentation do we follow?
https://google.github.io/styleguide/shellguide.html#indentation

This looks similar: https://wiki.bash-hackers.org/scripting/style?s[]=indent#indentation_guidelines

Gitlab suggests in https://docs.gitlab.com/ee/development/shell_scripting_guide/#formatting to run `shfmt` like this: 
```
shfmt -i 2 -ci -w scripts/**/*.sh
```
 
What's your preference?

Reply:
I think someone already choose 4 (https://github.com/DOMjudge/domjudge/commit/1eb396cf02c2f079c373423dcd9c98acafe8abae) so the intent was that we don't have tabs except for Makefiles and 4 as indentation level (if I remember correctly)

Reply:
No 2-space indent, please! `-ci` should be included and `-kp` also if it otherwise touches any spacing that is not part of initial indentation/alignment.

Reply:
I did now run shfmt but it did format some things weirdly (like `done` with four spaces less than `do`) and I undid them manually. It didn't change though what you requested. I'll merge this and we can discuss in person how to handle this properly.